### ISSUE TITLE
Update Documentation

### ISSUE THREAD

**Issue by jniclas** [ivanwick/bekantfirmware#9]
Hey, 
I use the firmware since two days successfully, works great 👌🏻 Thank you very much for developing it.
I followed the instruction "[PICkit 3 programmer and MPLAB IPE on Linux](https://github.com/ivanwick/bekantfirmware/wiki/PICkit3)" but with a Win 11 PC. So you could extend the documentation, that it is also possible to use the PICkit3 with Win 11 👍🏻 
And I found it hard to find the download page for the MPLAP IPE software. You could add a link to https://www.microchip.com/en-us/tools-resources/develop/mplab-x-ide in the same guide?

**Comment by ivanwick** [comment]
Excellent! Thank you for reporting your experience flashing the firmware using the IPE on Windows 11.

I updated the documentation on [PICkit3](https://github.com/ivanwick/bekantfirmware/wiki/PICkit3) with the MPLAB download link and mentioned that it is available on Linux, Windows, and macOS. This is a nice improvement.

By the way, when I was looking at the downloads, I also found the  [PICkit 3 Stand-Alone Programmer App v1.0](https://ww1.microchip.com/downloads/en/DeviceDoc/PICkit_3_Programmer_1_0_Setup_A.zip) available from the [Download Archives](https://www.microchip.com/en-us/tools-resources/archives/mplab-ecosystem). Did you try this stand-alone app?

If it works, I would recommend the stand-alone app for people with a PICkit3 using Windows, because it is a lot smaller to download and simpler than installing the whole MPLAB IDE and IPE. I don't have a PICkit3 device so I can't test it myself, but maybe you did.
